### PR TITLE
Make a coverage

### PR DESCRIPTION
**Pull request by arufanov**
Jacoco has been added

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -121,6 +122,45 @@
...
+                        </goals>
+                        <configuration>
+                            <rules>
+                                <rule>
+                                    <element>PACKAGE</element>
+                                    <limits>
+                                        <limit>
+                                            <counter>LINE</counter>
+                                            <value>COVEREDRATIO</value>
+                                            <minimum>0.50</minimum>
```
Comment (yurichervoniy) [inline pom.xml:155]: I think we need to discuss that value

File: src/test/java/com/a5lab/tabr/TabrApplicationTests.java [hunk 2]
```diff
@@ -10,6 +10,9 @@ class TabrApplicationTests {
 
     @Test
     void contextLoads() {
+        int a = 1;
```
Comment (yurichervoniy) [inline src/test/java/com/a5lab/tabr/TabrApplicationTests.java:13]: Very strange test =)

Comment (arufanov) [reply src/test/java/com/a5lab/tabr/TabrApplicationTests.java]: @yurichervoniy 
* I have removed it

File: pom.xml (hunk 1 above)
Comment (arufanov) [reply pom.xml]: Current coverage is 15, I have setup 14 to not block.

**Comment by arufanov** [comment]
@yurichervoniy 
fixed,
* report is created during test phase
* check and block coverage limit occurs during prepare-package phase.

File: pom.xml [hunk 3]
```diff
@@ -121,6 +122,50 @@
...
+                        <goals>
+                            <goal>report</goal>
+                        </goals>
+                    </execution>
+                    <execution>
+                        <id>jacoco-check</id>
+                        <goals>
+                            <goal>check</goal>
+                        </goals>
+                        <phase>prepare-package</phase>
```
Comment (yurichervoniy) [inline pom.xml:151]: I can skip test?

**Review by yurichervoniy — APPROVED** [review]

**Pull request merged**